Documentation/platforms/sim/sim/boards/sim/index.rst

```diff
@@ -1886,6 +1886,22 @@ NOTES:
   you will find this annoying.  You can disable the password protection
   by de-selecting CONFIG_NSH_CONSOLE_LOGIN=y.
 
+can
+---
+
+This is a configuration with simulated CAN support. Both CAN character driver
+and SocketCAN are enabled and use the host ``vcan0`` interface.
+The `vcan0` host interface must be available when NuttX is started.
+
+The SocketCAN interface on NuttX must be up before use::
+
+  ifup can0
+
+For the CAN character device, there is ``examples/can`` application enabled in
+read-only mode.
+
+Additionally, SocketCAN ``candump`` and ``cansend`` utils are enabled.
+
```

Comment:
Please include basic info about what need to be done on Linux side to get Virtual CAN working or some link to a page where it is explained

Reply:
+1 :-)